bundle-workflow/src/run_integ_test.py

```diff
@@ -29,13 +29,16 @@
 def parse_arguments():
     parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle")
     parser.add_argument(
-        "--bundle-manifest", type=argparse.FileType("r"), help="Bundle Manifest file."
```

Comment:
We need another parameter for test run id.

Reply:
I added that in my PR. See #467